proposals/csharp-8.0/default-interface-methods.md

```diff
@@ -741,12 +748,18 @@ but `C` is not recompiled. What happens when the program is run? An invocation o
 
 ### Permit `partial` in interface? (closed)
 
+This was not implemented in C# 8.
```

Comment:
I am not sure about this. I see a lot of DIM tests using partial interfaces. If partial interfaces were not supported prior to DIM support, it looks like they become available at the same time.

Reply:
Thanks @AlekseyTs   

I see what I had done wrong:  I created an interface that declared partial methods with arguments and different return types. Those weren't supported in C# 8. 

Fixed.